testing/test_assertion.py

```diff
@@ -1664,7 +1674,9 @@ def test_raising_repr():
     """
     )
     result = pytester.runpytest()
-    result.stdout.fnmatch_lines(["E       AssertionError: <exception str() failed>"])
+    result.stdout.fnmatch_lines(
+        ["E       AssertionError: <unprintable AssertionError object>"]
```

Comment:
This change should be reverted it seems.

Reply:
Right, it's strange I had an issue locally.